src/gui/settingswindow.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * @file settingswindow.h
+ * @brief Header file for the settings GUI page
+ * @copyright Copyright (C) 2021  Andrew Ritchie
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef SRC_GUI_SETTINGSWINDOW_H_
+#define SRC_GUI_SETTINGSWINDOW_H_
+
+namespace Ui {
```

Comment:
Is the `Ui` namespace used anywhere? Shouldn't this be `GUI` and I think the class needs to be defined within the namespace for Doxygen to generate the documentation correctly (like I changed `main window.h` to be)

Reply:
No problem, changed it to GUI.